gallery/lib/home.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/material.dart';
+
+import 'demo.dart';
+import 'settings.dart';
+import 'splash.dart';
+
+class HomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Gallery'),
+        leading: IconButton(icon: Icon(Icons.color_lens), onPressed: () {
+          Navigator.of(context).push(
+            MaterialPageRoute(
+              fullscreenDialog: true,
+              builder: (context) => SplashPage(),
+            ),
+          );
+        }),
```

Comment:
nit: trailing comma and reformat here I think.

Reply:
Done